lib/user_agent/operating_systems.rb

```diff
@@ -1,6 +1,7 @@
 class UserAgent
   module OperatingSystems
-    IOS_VERSION_REGEX = /CPU (?:iPhone |iPod )?OS ([\d_]+) like Mac OS X/
+    ANDROID_VERSION_REGEX = /[Aa]ndroid v?([\d\.]+)/
```

Comment:
```suggestion
    ANDROID_VERSION_REGEX = /[Aa]ndroid v?(?<version>[\d\.]+)/
```
Can we do a named group here and then below, in `def self.normalize_android` (line 27) do something like:

```
if os.match(ANDROID_VERSION_REGEX)
  if match[:version].nil?
    'Android'
...
```